sensing/radar_scan_to_pointcloud2/schema/radar_scan_to_pointcloud2.schema.json

```diff
@@ -0,0 +1,35 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for Radar Scan to PointCloud2 Node",
+  "type": "object",
+  "definitions": {
+    "radar_scan_to_pointcloud2": {
+      "type": "object",
+      "properties": {
+        "publish_amplitude_pointcloud": {
+          "type": "boolean",
+          "description": "Whether publish radar pointcloud whose intensity is amplitude.",
+          "default": true
+        },
+        "publish_doppler_pointcloud": {
+          "type": "boolean",
+          "description": "Whether publish radar pointcloud whose intensity is doppler velocity.",
+          "default": false
+        }
+      },
+      "required": ["publish_amplitude_pointcloud", "publish_doppler_pointcloud"]
```

Comment:
New lines between parameters.
```
"required": [
  "publish_amplitude_pointcloud",
  "publish_doppler_pointcloud"
]
```
I can see in
* https://autowarefoundation.github.io/autoware-documentation/main/contributing/coding-guidelines/ros-nodes/parameters/#json-schema

that there is no line when there are two parameters. This should be updated!

Reply:
Got it. Done.

Reply:
Aha, that's the same issue as for the documentation. I guess this will be fine! Sorry about the inconvenience.